k9mail/src/main/java/com/fsck/k9/message/quote/TextQuoteCreator.java

```diff
@@ -0,0 +1,83 @@
+package com.fsck.k9.message.quote;
+
+
+import android.content.res.Resources;
+
+import com.fsck.k9.Account.QuoteStyle;
+import com.fsck.k9.R;
+import com.fsck.k9.helper.Utility;
+import com.fsck.k9.mail.Address;
+import com.fsck.k9.mail.Message;
+import com.fsck.k9.mail.Message.RecipientType;
+import com.fsck.k9.mail.MessagingException;
+
+import static com.fsck.k9.message.quote.QuoteHelper.QUOTE_BUFFER_LENGTH;
+
+
+public class TextQuoteCreator {
+    private static final int REPLY_WRAP_LINE_WIDTH = 72;
+
+    /**
+     * Add quoting markup to a text message.
+     * @param originalMessage Metadata for message being quoted.
+     * @param messageBody Text of the message to be quoted.
+     * @param quoteStyle Style of quoting.
+     * @return Quoted text.
+     * @throws MessagingException
+     */
+    public static String quoteOriginalTextMessage(Resources resources, Message originalMessage, String messageBody, QuoteStyle quoteStyle, String prefix) throws MessagingException {
```

Comment:
None of the code was changed. It was only moved around. Cleaning this is up and adding tests is a separate task.